components/swatchgroup/index.css

```diff
@@ -10,21 +10,32 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+.spectrum-SwatchGroup {
+  --spectrum-swatchgroup-spacing-compact: var(--spectrum-spacing-50);
+  --spectrum-swatchgroup-spacing-regular: var(--spectrum-spacing-75);
+  --spectrum-swatchgroup-spacing-spacious: var(--spectrum-spacing-100);
+}
+
 .spectrum-SwatchGroup {
   display: inline-flex;
   flex-direction: row;
   flex-wrap: wrap;
   align-items: flex-start;
   justify-content: flex-start;
+}
+
+/* Regular (Default) */
+.spectrum-SwatchGroup {
 
```

Comment:
Sorry I missed this before, extra space here on line 29.

Reply:
it's not too bad :) not such a big deal